base/applications/cmdutils/more/lang/tr-TR.rc

```diff
@@ -33,7 +33,7 @@ Options:\n\
     files    List of files to display, separated by blanks.\n\
 \n\
 When Extended Features are enabled, the following commands are available\n\
-at the ""-- Continue --"" prompt:\n\
+at the ""-- Sürdür --"" prompt:\n\
```

Comment:
The more command was updated since then, so please review introduced changes and translate missing parts.

Reply:
okey. I will review and translate missing parts with Can as soon as possible.

Reply:
All issues resolved.